client/radial/radial.js

```diff
@@ -0,0 +1,120 @@
+function mountRadial() {
+    "use strict"
+
+  var width = document.body.clientWidth;
+  var height = document.body.clientHeight;
+  var tree;
+  var min_zoom = 0.1;
+  var max_zoom = 7;
+  var zoom = d3v3.behavior.zoom().scaleExtent([min_zoom,max_zoom])
+
+  var force = d3v3.layout.force()
+    .charge(-800)
+    .size([width, height]);
+
+  var svg = d3v3.select("#graph").append("svg")
+    .attr("width", width)
+    .attr("height", height);
+
+  const render = (data) => {
+    // console.log(data);
+    const arr = [];
+
+    for (let i in data) {
+      if (i === 'root') data[i]['depth'] = 0;
+      else data[i]['depth'] = 1;
+      arr.push(data[i]);
+    }
+    return arr;
+  };
+
+  tree = {
+    nodes: render(data),
+    links: getLinks(render(data))
+  };
+  console.log(tree);
+  updateForce();
```

Comment:
despite the fact that order doesn't really matter in JS, it's a good practice to place all function declarations in the same place in the file (usually the beggining) and then call them when they need to be call. Try to avoid declaring functions after they are called.